Cloudinary-pwa-kit/packages/pwa-cloudinary/app/components/cloudinary-image-gallery/index.jsx

```diff
@@ -23,10 +23,10 @@ const EnterKeyNumber = 13
  */
 const CloudinaryImageGallery = ({ size, cloudinaryImageGallery = {} }) => {
     const [selectedIndex, setSelectedIndex] = useState(0)
+    const [imageUrls, setImageUrls] = useState('')
     const styles = useMultiStyleConfig('ImageGallery', { size })
-    var imageUrls
     if (!cloudinaryImageGallery.galleryEnabled) {
-        imageUrls = cloudinaryImageGallery.images.imageURLs
+        setImageUrls(cloudinaryImageGallery?.images?.imageURLs)
     }
     const imageUrl = imageUrls ? imageUrls?.[selectedIndex] : null
```

Comment:
Previously, the 'imageUrls' was 'cloudinaryImageGallery?.images?.imageURLs' and now it comes from "useState('')".
Is this intentional?

Reply:
yes, this is